geopandas/base.py

```diff
@@ -189,6 +189,9 @@ def unary_geo(self, op):
         x = vectorized.geo_unary_op(op, self._geometry_array.data)
         return GeoSeries(x, index=self.index, crs=self.crs)
 
+    def coords(self):
+        return vectorized.get_coordinates(self._geometry_array.data)
```

Comment:
But, then the question is: should the result here be wrapped in a Series ? 
For consistency with other methods, that is most logical. Although putting tuples inside a Series is not always recommended.

(anyway for internal use for plotting, we can directly call `vectorized.get_coordinates`)